monkey/infection_monkey/monkey.py

```diff
@@ -114,14 +111,12 @@ def __init__(self, args):
 
         self._singleton = SystemSingleton()
         self._opts = self._get_arguments(args)
-        self._server_strings = [str(s) for s in self._opts.servers]
 
         self._agent_event_serializer_registry = self._setup_agent_event_serializers()
 
         server, self._island_api_client = self._connect_to_island_api()
-        # TODO: `address_to_port()` should return the port as an integer.
-        self._cmd_island_ip, self._cmd_island_port = address_to_ip_port(server)
-        self._cmd_island_port = int(self._cmd_island_port)
+        self._cmd_island_ip = server.ip
+        self._cmd_island_port = server.port
```

Comment:
What's the point of spliting this up then joining it back to socket address? I don't think we need these lines

Reply:
Some code (`VictimHostFactory`, logging, maybe more) ahead need the IP and port separately. The "joining it back to socket address" has been fixed now but we do need to split them up.